packages/views/screens/managers/register/register.js

```diff
@@ -1,79 +1,51 @@
-import React from 'react';
+import React, { useState, useEffect, useCallback } from 'react';
 import grid from 'flexboxgrid/dist/flexboxgrid.css';
-import { generatePassphrase } from '@views/utilities/passphrase';
-import { extractAddressFromPassphrase } from '@wallet/utilities/account';
 import routes from '@screens/router/routes';
 import MultiStepProgressBar from '@shared/multiStepProgressBar';
 import MultiStep from '@dpos/manage/registerDelegateMultiStep';
+import useCreateAccounts from '@wallet/hooks/useCreateAccounts';
 import ChooseAvatar from './chooseAvatar';
 import BackupPassphrase from './backupPassphrase';
 import ConfirmPassphrase from './confirmPassphrase';
 import AccountCreated from './accountCreated';
 import styles from './register.css';
 
-class Register extends React.Component {
-  constructor() {
-    super();
-    this.state = {
-      accounts: [],
-      selectedAccount: {},
-    };
+const Register = ({ account, token, history }) => {
+  const [selectedAccount, setSelectedAccount] = useState({});
+  const { accounts } = useCreateAccounts();
```

Comment:
it may better to rename it as suggestionAccounts

Reply:
okay